deploy/blue-green-deployment.ts

```
@@ -0,0 +1,564 @@
+
+  if (!command) {
+    console.error('Command required. Available commands: deploy, status, rollback');
+    process.exit(1);
+  }
+
+  const deployment = new BlueGreenDeployment();
+
+  try {
+    switch (command) {
+      case 'deploy':
+        const version = args[1] || `v${Date.now()}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)